src/App.php

```diff
@@ -198,6 +199,16 @@ function () use ( $slim ) {
 					$page( 'callback' );
 				} )->name( 'oauth_callback' );
 			} );
+		$slim->get( '/index.css',
```

Comment:
Since you're fetching '/index.css', it makes more sense for you to put it under group('/') - Line 157 and just fetch 'index.css'.